src/pages/Patient/index.tsx

```diff
@@ -37,19 +52,27 @@ export const PatientsPage = () => {
       </Box>
       <PatientModal
         open={isModalOpen}
-        onClose={() => setIsModalOpen(false)}
+        onClose={() => {
+          setIsModalOpen(false);
+          setPatient(initPatient);
+        }}
         onUpdate={() => refetch()}
+        patient={patient}
+        setPatient={(patient) => setPatient(patient)}
       />
       <GridSearchFilter
         search={search}
         setSearch={setSearch}
         applyFilters={() => setSearchValue(search)}
       />
       <PatientsTable
+        isModalOpen={isModalOpen}
+        setIsModalOpen={setIsModalOpen}
         data={data}
         isLoading={isLoading}
         paginationModel={paginationModel}
         setPaginationModel={setPaginationModel}
```

Comment:
from table  You  will  alway  have  data  ...   so You  can write shorter  onSelectPatient={setPatient}